schema/.markdownlint.jsonc

```diff
@@ -139,7 +139,7 @@
 
   // MD026/no-trailing-punctuation - Trailing punctuation in heading
   "MD026": {
-    // Punctuation characters
+    // Punctuation characters to remove
```

Comment:
Thanks! To be picky, it's only removed when fixing. What about using the word "block" or "prevent" instead?

Reply:
Also, any reason not to do the same improvement for the other rule?

Reply:
I was less clear on the use in MD036 when I was reading the code, so I wasn't sure about the wording.
I'll take another cut at the wording on this one though